src/turn.erl

```diff
@@ -74,7 +74,7 @@
          rcvd_bytes = 0 :: non_neg_integer(), rcvd_pkts = 0 :: non_neg_integer(),
          sent_bytes = 0 :: non_neg_integer(), sent_pkts = 0 :: non_neg_integer(),
          parent :: pid() | undefined, parent_resolver :: function() | undefined,
-         start_timestamp = get_timestamp() :: integer(),
+         port_black_list = [] :: list(), start_timestamp = get_timestamp() :: integer(),
```

Comment:
Maybe call this `unknonw_ports` and use set instead of list https://www.erlang.org/doc/man/sets.html